.github/pull_request_template.md

```diff
@@ -3,6 +3,20 @@
  <!-- please add an icon to the title of this PR (see ../CONTRIBUTING.md) -->
```

Comment:
if we drop the icons we should also update CONTRIBUTING.md to match instructions in here